bin/specification_urls.json

```diff
@@ -0,0 +1,34 @@
+{
+    "draft3": {
+        "core": "https://json-schema.org/draft-03/draft-zyp-json-schema-03.pdf"
+    },
+    "draft4": {
+        "core": "https://json-schema.org/draft-04/draft-zyp-json-schema-04#rfc.section.",
+        "validation": "https://json-schema.org/draft-04/draft-fge-json-schema-validation-00#rfc.section.",
+        "hyper-schema": "https://json-schema.org/draft-04/draft-luff-json-hyper-schema-00#rfc.section."
+    },
+    "draft6": {
+        "core": "https://json-schema.org/draft-06/draft-wright-json-schema-01#rfc.section.",
+        "validation": "https://json-schema.org/draft-06/draft-wright-json-schema-validation-01#rfc.section.",
+        "hyper-schema": "https://json-schema.org/draft-06/draft-wright-json-schema-hyperschema-01#rfc.section."
+    },
+    "draft7": {
+        "core": "https://json-schema.org/draft-07/draft-handrews-json-schema-01#rfc.section.",
+        "validation": "https://json-schema.org/draft-07/draft-handrews-json-schema-validation-01#rfc.section.",
+        "hyper-schema": "https://json-schema.org/draft-07/draft-handrews-json-schema-hyperschema-01#rfc.section."
+    },
+    "draft2019-09": {
+        "core": "https://json-schema.org/draft/2019-09/draft-handrews-json-schema-02#rfc.section.",
+        "validation": "https://json-schema.org/draft/2019-09/draft-handrews-json-schema-validation-02#rfc.section.",
+        "hyper-schema": "https://json-schema.org/draft/2019-09/draft-handrews-json-schema-hyperschema-02#rfc.section."
+    },
+    "draft2020-12": {
+        "core": "https://json-schema.org/draft/2020-12/draft-bhutton-json-schema-01#section-",
+        "validation": "https://json-schema.org/draft/2020-12/draft-bhutton-json-schema-validation-01#section-",
+        "hyper-schema": "https://json-schema.org/draft/2019-09/draft-handrews-json-schema-hyperschema-02#rfc.section."
+    },
+    "ecma262": "https://262.ecma-international.org/",
+    "perl5": "https://perldoc.perl.org/perlre#",
+    "rfc": "https://www.rfc-editor.org/rfc/",
+    "iso": "https://www.iso.org/obp/ui"
+}
```

Comment:
Please configure your editor to make sure all files end in a newline or be sure you do so manually, one is missing here.